resolwe_bio/processes/workflows/wes.yml

```diff
@@ -3,7 +3,7 @@
   data_name: "{{ reads|sample_name|default('?') }}"
   requirements:
     expression-engine: jinja
-  version: 1.0.0
+  version: 1.0.1
```

Comment:
Since you have removed the parameter `md_validation_stringency` this is a breaking change and should result in the bump of first version. So `2.0.0`